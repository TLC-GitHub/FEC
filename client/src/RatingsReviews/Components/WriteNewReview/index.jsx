import React from 'react';


function WriteNewReview() {
  return (
    <div>
      Write New Review
    </div>
  )
}

export default WriteNewReview;