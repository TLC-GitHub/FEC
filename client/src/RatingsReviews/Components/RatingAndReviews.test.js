import React from 'react';
import renderer from 'react-test-renderer';
import WriteNewReview from './WriteNewReview/index.jsx'


describe('Ratings and Reviews', () => {
  it('Renders Ratings and Reviews components to the DOM', () => {
    const component = renderer.create(<WriteNewReview />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});