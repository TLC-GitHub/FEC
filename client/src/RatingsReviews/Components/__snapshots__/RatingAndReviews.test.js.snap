// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ratings and Reviews Renders Ratings and Reviews components to the DOM 1`] = `
<div>
  Write New Review
</div>
`;
